apps/allocations/app/app-state-reducer.js

```diff
@@ -43,7 +45,10 @@ function appStateReducer(state) {
       ...budget,
       active: budget.hasBudget && Number(budget.amount) > 0,
       // get some extra info about the token
-      token: getTokenFromAddress(budget.token, balances)
+      token: getTokenFromAddress(budget.token, balances),
+      // if current period end date is less than now, current period is over
+      // but not updated on chain. so we reset the remaining budget
+      remaining: endDate < currentDate ? budget.amount : budget.remaining,
```

Comment:
Just wondering if this will be intuitive for the user, I think the "remaining" field with abruptly change with the condition transition, but not sure if we should show also some visual indicator indicating the budget or period reset

Reply:
I think that for now we don't have to get fancy with a transition, can log an enhancement for the future though? It's rare that people will be staring at the page right when the transition occurs, and would prefer to avoid churn on getting this feature out